lib/SBOLDocument.js

```diff
@@ -52,11 +52,15 @@ var ModuleDefinition = require('./ModuleDefinition'),
     Cut = require('./Cut'),
     GenericLocation = require('./GenericLocation'),
     GenericTopLevel = require('./GenericTopLevel'),
+    CombinatorialDerivation = require('./CombinatorialDerivation'),
+    VariableComponent = require('./VariableComponent'),
     ProvActivity = require('./ProvActivity'),
     ProvAssociation = require('./ProvAssociation'),
     ProvAgent = require('./ProvAgent'),
     ProvPlan = require('./ProvPlan'),
     ProvUsage = require('./ProvUsage')
+    CombinatorialDerivation = require('./CombinatorialDerivation'),
```

Comment:
no big deal, but we should remove the extra imports of Combinatorial and Variable